gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
+networkTimeout=10000
```

Comment:
Unsure about the changes to the gradle files. How come they happened? I would prefer if they were in a separate commit.

Reply:
It's part of the gradle wrapper upgrade to v8.1.1 (latest) as was suggested in the IntelliJ plugin dev docs (to always use latest cradle version).

Reply:
I meant that I didn't change those files manually. it was the gradle wrapper upgrade script that did it.

Reply:
Here's some background: https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper